xil-Town05/cdasim_config/carma-carla-integration/sensors.json

```diff
@@ -0,0 +1,167 @@
+{
+    "sensors": [
+        {
+            "type": "sensor.camera.rgb",
+            "id": "follow",
+            "x": -5.0, "y": 0.0, "z": 5, "roll": 0.0, "pitch": -45.0, "yaw": 0.0,
+            "image_size_x": 800,
+            "image_size_y": 600,
+            "fov": 120.0,
+            "sensor_tick": 0.05,
+            "gamma": 2.2,
+            "shutter_speed": 200.0,
+            "iso": 100.0,
+            "fstop": 8.0,
+            "min_fstop": 1.2,
+            "blade_count": 5,
+            "exposure_mode": "histogram",
+            "exposure_compensation": 0.0,
+            "exposure_min_bright": 7.0,
+            "exposure_max_bright": 9.0,
+            "exposure_speed_up": 3.0,
+            "exposure_speed_down": 1.0,
+            "calibration_constant": 16.0,
+            "focal_distance": 1000.0,
+            "blur_amount": 1.0,
+            "blur_radius": 0.0,
```

Comment:
@chengyuan0124 Do you know if there are settings to get better video resolution for the rgb camera. Sometimes it is blurry

Reply:
The image_size_x and image_size_y seems to be the resolution. We can increase it to 1920x1080? @paulbourelly999 
Line 7,8